### ISSUE TITLE
[feature] Easy click to add sender to contacts list

### ISSUE THREAD

**Issue by alexzeitgeist** [RainLoop/rainloop-webmail#645]
I don't want to automatically add recipients to my contact list. But when I add a contact, it would be nice if I didn't had to enter the data manually. Instead, it would be nice to have an option to click on to add the sender of a mail to the contacts list. For instance, it could be placed in the top-right drop-down button (Reply, Forward, Edit as New, ...).

Cheers

**Comment by justdaj** [comment]
I agree -- I'd like to see a + symbol beside each address on a message.  I suggested this on the 'standard' version a few months ago but so far nothing.

**Comment by mat-m** [comment]
I do agree.

**Comment by derStephan** [comment]
+1

**Comment by verilucis** [comment]
+1

Thanks!

**Comment by jfding** [comment]
+1
will be very helpful

**Comment by rafny** [comment]
+1
